server/notebooks/views.py

```diff
@@ -31,13 +31,16 @@ def notebook_view(request, pk):
     notebook = get_object_or_404(Notebook, pk=pk)
     if "revision" in request.GET:
         revision = get_object_or_404(NotebookRevision, pk=int(request.GET["revision"]))
+        latest_revision_id = notebook.revisions.values_list("id", flat=True).first()
```

Comment:
I didn't check but I'm assuming this has an ordering on the model class so the first `id` is the most recent? There's also `notebook.revisions.get_latest_by("created").id` which explicitly provides the date field.

Reply:
that's correct on the ordering front :) i think your solution makes this more explicit, so i'll switch to that approach